src/oc/web/actions/activity.cljs

```diff
@@ -239,13 +242,30 @@
       (dis/dispatch! [:modal-editing-activate]))
     (dis/dispatch! [:modal-editing-deactivate])))
 
+
+(declare entry-save)
+
 (defn entry-save-on-exit
-  [edit-key activity-data entry-body]
+  [edit-key activity-data entry-body section-editing]
   (let [entry-map (assoc activity-data :body entry-body)
         cache-key (get-entry-cache-key (:uuid activity-data))]
     (uc/set-item cache-key entry-map
      (fn [err]
        (when-not err
+         (when (and (not= "published" (:status activity-data))
+                    (:has-changes activity-data)
+                    (not (:auto-saving activity-data)))
+           ;; dispatch that you are auto saving
+           (dis/dispatch! [:update [edit-key]
+                           #(merge % activity-data {:auto-saving true})])
+           (entry-save edit-key activity-data section-editing
+             (fn [entry-data-saved edit-key-saved {:keys [success body status]}]
+               (when success
+                 (remove-cached-item (:uuid activity-data))
```

Comment:
If you remove the cached item every time you save the local cookie won't be saving anymore. You need to remove the cache only in the case i described above.